frontend/public/components/graphs/area.tsx

```diff
@@ -110,13 +114,14 @@ export const AreaChart: React.FC<AreaChartProps> = ({
 
   const multiLine = processedData?.length > 1;
 
+  const legendData = processedData.map((d) => ({
+    childName: d[0].description,
+    name: d[0].description,
+    symbol: d[0].symbol,
+  }));
```

Comment:
This would affect all graphs rather introduce some prop something like `showLegend`

Reply:
@bipuladh Not sure if I'm understanding this suggestion correctly, but just to mention, this was actually taken out from [here](https://github.com/openshift/console/pull/10470/files#diff-5b35532236929db6e3f01d77c28b4939ab5ba00f9a2400b385723839a9d053adL115-L119) and introduced in the global scope so it could be accessible from [here](https://github.com/openshift/console/pull/10470/files#diff-5b35532236929db6e3f01d77c28b4939ab5ba00f9a2400b385723839a9d053adR173).